src/transaction_builder.js

```diff
@@ -216,6 +216,8 @@ function expandOutput (script, ourPubKey) {
 }
 
 function prepareInput (input, ourPubKey, redeemScript, witnessValue, witnessScript) {
+  redeemScript = redeemScript || input.redeemScript
```

Comment:
Not much to it,  but, did we decide to *enforce* users providing the redeem script rather than signing it without them knowing?